### PR TITLE
cncf#631 white_list_helm_container_names changed to allow_list_helm_c…

### PR DESCRIPTION
**Pull request by Palerion**
## Description
white_list_helm_container_names changed to allow_list_helm_container_names to better conform with inclusivity standards.

## Issues:
Refs: #631

## How has this been tested:
 - [X] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [X] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [X] Tasks in issue are checked off

### CODE REVIEW

**Comment by Palerion** [comment]
My apologies, I'm relatively new to this, I'm unsure what's gone wrong with the checks.

**Comment by agentpoyo** [comment]
> My apologies, I'm relatively new to this, I'm unsure what's gone wrong with the checks.

That's expected as it requires credentials for github actions.

File: src/tasks/utils/config.cr [hunk 1]
```diff
@@ -75,9 +75,9 @@ module CNFManager
       end
       helm_chart_path = destination_cnf_dir + "/" + working_chart_directory 
       manifest_file_path = destination_cnf_dir + "/" + "temp_template.yml"
-      white_list_container_names = optional_key_as_string(config, "white_list_helm_chart_container_names")
-      if config["white_list_helm_chart_container_names"]?
-        white_list_container_names = config["white_list_helm_chart_container_names"].as_a.map do |c|
+      white_list_container_names = optional_key_as_string(config, "allowlist_helm_chart_container_names")
```
[the remark this answers is not in the document]
Comment (lixuna) [reply src/tasks/utils/config.cr:78]: @agentpoyo eventually I think that, over time, all references will need to be renamed. Would you like to create a new issue for renaming the remaining areas?

[the remark this answers is not in the document]
Comment (agentpoyo) [reply src/tasks/utils/config.cr:78]: @lixuna  I can do that.

**Pull request merged**